### DISCUSSION TITLE
Regarding the markdown plugin, is it possible add the title from the same file?

### DISCUSSION THREAD

**Discussion by ekiim** [vikejs/vike#1654] [Discussion]
I know that to define the "metadata" around the file we need to define more files sibling to the markdown file that matches the page, would it be possible to use something like a Yaml metadata block or export a json object from it such that it functions as the "metadata" for the page?

By metadata, I mean, stuff that I can put in the `head` section or as a title.

My aim is for sites working with react and doing SSG.

**Comment by brillout** [comment]
Have you seen https://vike.dev/markdown#metadata and https://vike.dev/markdown#with-frontmatter?
